packages/world/ts/config/v2/codegen.ts

```diff
@@ -0,0 +1,12 @@
+import { isObject, mergeIfUndefined } from "@latticexyz/store/config/v2";
+import { CODEGEN_DEFAULTS } from "./defaults";
```

Comment:
we should decide if we wanna SHOUT our constants or not, feels not necessary in today's world with `const`

Reply:
no strong opinion here